testing/integration/otel_test.go

```diff
@@ -1099,3 +1099,168 @@ service:
 	fixtureWg.Wait()
 	require.True(t, err == nil || errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded), "Retrieved unexpected error: %s", err.Error())
 }
+
+func TestOtelFBReceiverBatchSplitE2E(t *testing.T) {
+	// This test asserts that the batcher configuration from the elasticseachexporter
+	// splits batches to avoid a 413 Request Entity Too Large error from Elasticsearch.
+	info := define.Require(t, define.Requirements{
```

Comment:
Just double checking why we need this to be an integration test that runs the entire collector. What additional test coverage is this adding? Why can't this be covered by a unit test in the exporter?

I like integration tests, they catch a lot of problems, but they are also driving our CI execution time to infinity :)

Reply:
Agree, I think this is better suited to be in the exporter. Actually, it might as well be covered in the exporter already by https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36396. I will double check.